cincinnati/src/lib.rs

```diff
@@ -1017,6 +1040,45 @@ pub mod testing {
 
         removed_metadata
     }
+
+    /// Replace the digests by versioned tags in the release payload strings.
+    pub fn payload_replace_sha_by_tag_fn(graph: &mut Graph) -> Fallible<()> {
+        graph
+            .iter_releases_mut(|ref mut release| {
+                match release {
+                    Release::Concrete(ref mut release) => {
+                        // replace digest by tag to match expectency
+                        let version = release.version.to_string();
+                        let source_front = release.payload.split('@').next().ok_or_else(|| {
+                            Error::msg(format!("invalid version string {:?}", version))
+                        })?;
+                        release.payload = format!("{}:{}", source_front, version);
+
+                        Ok(())
+                    }
+                    _ => panic!("should not get here"),
+                }
+            })
+            .context("replacing the sha by the tag in the payload string")
+    }
+
+    /// Remove the registry and the repository in release payload strings.
+    pub fn payload_remove_registry_and_repo(graph: &mut Graph) -> Fallible<()> {
+        graph
+            .iter_releases_mut(|ref mut release| {
+                match release {
+                    Release::Concrete(ref mut release) => {
+                        // replace digest by tag to match expectency
+                        let version = release.version.to_string();
+                        release.payload = version;
+
+                        Ok(())
+                    }
+                    _ => panic!("should not get here"),
```

Comment:
Another console message which I think should be modified to be more meaningful. Forgot to point this out in last review.